app/models/site.rb

```diff
@@ -203,10 +203,24 @@ def self.clear_anon_cache!
     MessageBus.publish(SITE_JSON_CHANNEL, '')
   end
 
+  def self.welcome_topic_banner_cache_key(user_id)
+    "show_welcome_topic_banner:#{user_id}"
+  end
+
   def self.show_welcome_topic_banner?(guardian)
     return false unless guardian.is_admin?
```

Comment:
I realise this is old code, but we should change this to `return false if !guardian.is_admin?` to appease Sam ;)

Reply:
Done :)